crates/rolldown_std_utils/src/pretty_type_name.rs

```diff
@@ -0,0 +1,30 @@
+use std::borrow::Cow;
+
+use regex::Regex;
+use std::sync::LazyLock;
+
+static MODULE_MATCHER_RE: LazyLock<Regex> = LazyLock::new(|| Regex::new(r"(?:\w+::)").unwrap());
```

Comment:
I wonder if we could remove the `regex` dependency and still print the type name in a pretty way.